custom_components/tuya_local/devices/warmehaus_AFD02TJ_thermostat_c.yaml

```diff
@@ -0,0 +1,76 @@
+name: WarmeHaus Thermostatic Towel Rail Electric Heating Element
+products:
+  - id: fgzmbjxhgkuwgrav
+    manufacturer: WarmeHaus
+    model: AFD-02-TJ
+entities:
+  - entity: climate
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: 'off'
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: cold
+                value: cool

```

Comment:
Does it really have a cool mode?

Below on hvac_action, there is also a condition mentioning auto, which does not appear here.

Reply:
sorted

Reply:
Have also added the DPS info to description.